From: Chow Loong Jin <hyperair@debian.org>
Date: Fri, 27 Jan 2012 05:18:44 +0800
Subject: Drop direct linkage of libzen in .pc

libzen is a private dependency, not a public dependency, so there is no need to
link against it directly in client code.
---
 Project/GNU/Library/configure.ac       |    1 -
 Project/GNU/Library/libmediainfo.pc.in |    2 +-
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/Project/GNU/Library/configure.ac b/Project/GNU/Library/configure.ac
index 88d4ed2..ecee8f0 100755
--- a/Project/GNU/Library/configure.ac
+++ b/Project/GNU/Library/configure.ac
@@ -397,7 +397,6 @@ else
 		else
 			with_zenlib="system"
 			LIBS="$LIBS $(pkg-config --libs libzen)"
-			MediaInfoLib_LIBS="$MediaInfoLib_LIBS $(pkg-config --libs libzen)"
 			MediaInfoLib_LIBS_Static="$MediaInfoLib_LIBS_Static $(pkg-config --variable=LIBS libzen)"
 		fi
 	else
diff --git a/Project/GNU/Library/libmediainfo.pc.in b/Project/GNU/Library/libmediainfo.pc.in
index 1ee9309..f9dfd8f 100755
--- a/Project/GNU/Library/libmediainfo.pc.in
+++ b/Project/GNU/Library/libmediainfo.pc.in
@@ -9,5 +9,5 @@ la_name=lib@MediaInfoLib_LibName@.la
 Name: libmediainfo
 Version: @PACKAGE_VERSION@
 Description: MediaInfoLib
-Libs: -L@libdir@ -l@MediaInfoLib_LibName@ -lzen -lz -lpthread
+Libs: -L@libdir@ -l@MediaInfoLib_LibName@ -lz -lpthread
 Cflags: -I@includedir@ @MediaInfoLib_CXXFLAGS@
